pkg/api/types.go

```diff
@@ -584,10 +584,28 @@ const (
 	ClaimLost PersistentVolumeClaimPhase = "Lost"
 )
 
+type HostPathType string
+
+const (
+	// If nothing exists at the given path, an empty directory will be created there. Otherwise, behaves like HostPathExists
+	HostPathUnset HostPathType = ""
```

Comment:
I would not even call this out.  Optional fields need to be pointers, so "" is not a valid value.